src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -723,38 +713,16 @@ private void SpeciesListMenuIndexChanged(int index)
         var speciesName = speciesListMenu.Popup.GetItemText(index);
         var species = speciesHistoryList[generationDisplayed].Values.First(p => p.FormattedName == speciesName);
 
-        if (species == speciesPreview.PreviewSpecies)
+        if (species == speciesDetailsPanel.PreviewSpecies)
```

Comment:
This check should be put in the PreviewSpecies property so that it skips applying the GUI control changes if the new value is the same as the old one.